core/src/main/java/org/apache/cxf/version/Version.java

```diff
@@ -29,19 +29,15 @@ public final class Version {
     private static String fullVersion;
     private static String buildNumber;
 
-    private static final String VERSION_BASE = "/org/apache/cxf/version/";
+    private static final String VERSION_BASE = "org/apache/cxf/version/";
```

Comment:
@AmshikaH thank you for looking into this, but I believe this change has high risks of breaking things. The CXF deals with many containers (notably, OSGi / Apache Karaf / Apache Felix) that have own classloaders and are sensitive to resource naming.

Reply:
@reta, noted, thank you for the feedback.